nestkernel/node_collection.h

```diff
@@ -194,6 +194,14 @@ class NodeCollection
    */
   static NodeCollectionPTR create( const TokenArray& node_ids );
 
+  /**
+   * Create a NodeCollection from a single node ID. Results in a primitive.
```

Comment:
`Results in a primitive`: Is this sentence complete?

Reply:
A primitive is a NodeCollection that is contiguous and homogeneous. It is used as a noun throughout the file, so if the wording is confusing, it probably has to be changed everywhere.

Reply:
Ah, I see. In that case it is best to leave ``primitive`` as it is.